recipes/setproctitle/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install
```

Comment:
Yeah...  It is really ugly (and inconsistent) to remove `build.sh` and leave `bld.bat`.